content/hi/docs/tutorials/_index.md

```diff
@@ -0,0 +1,65 @@
+---
+title: ट्यूटोरियल
+main_menu: true
+no_list: true
+weight: 60
+content_type: concept
+---
+
+<!-- overview -->
+
+प्रलेखन के इस खंड में ट्यूटोरियल हैं।
+ट्यूटोरियल दिखाता है कि किसी एकल कार्य से बड़े [लक्ष्य](/docs/tasks/) को कैसे पूरा किया जाए।
+आमतौर पर एक ट्यूटोरियल में कई खंड होते हैं,
+जिनमें से प्रत्येक में चरणों के क्रम होते हैं।
+प्रत्येक ट्यूटोरियल से परिचित होने से पहले, हम आपको 
+[मानकीकृत शब्दावली](/docs/reference/glossary/) पृष्ट को बुकमार्क करने की सलाह देते हैं। 
+<!-- body -->
+
+## मूलभूत
+
+* [मूलभूत कुबरनेट्स](/hi/docs/tutorials/kubernetes-basics/) एक गहन इंटरैक्टिव ट्यूटोरियल है जो आपको कुबेरनेट्स प्रणाली को समझने और कुबेरनेट्स की कुछ बुनियादी सुविधाओं को आज़माने में मदद करता है।
+
+* [Introduction to Kubernetes (edX)](https://www.edx.org/course/introduction-kubernetes-linuxfoundationx-lfs158x#)
+
+* [Hello Minikube](/hi/docs/tutorials/hello-minikube/)
+
+## विन्यास
+
+* [Example: Configuring a Java Microservice
+](/docs/tutorials/configuration/configure-java-microservice/) (उदाहरण: Java माइक्रोसर्विस का विन्यास करना)
+
+* [Configuring Redis Using a ConfigMap](/docs/tutorials/configuration/configure-redis-using-configmap/) (कॉन्फिग मैप का उपयोग करके Redis का विन्यास करना)
+
+## स्टेटलेस एप्लीकेशन
+
+
+* [Exposing an External IP Address to Access an Application in a Cluster](/docs/tutorials/stateless-application/expose-external-ip-address/) (किसी क्लस्टर में किसी एप्लिकेशन तक पहुंचने के लिए बाहरी आईपी पते को उजागर करना)
+
+* [Example: Deploying PHP Guestbook application with Redis](/docs/tutorials/stateless-application/guestbook/) (उदाहरण: रेडिस के साथ PHP गेस्टबुक एप्लिकेशन को उपयोग करना)
+
+## स्टेटफुल एप्लीकेशन
+
+* [स्टेटफुलसेट मूलभूत](/docs/tutorials/stateful-application/basic-stateful-set/)
+
+* [उदाहरण: दृढ़ वॉल्यूम के साथ वर्डप्रेस और MySQL](/docs/tutorials/stateful-application/mysql-wordpress-persistent-volume/)
+
+* [उदाहरण: कैसेंड्रा को स्टेटफुल सेट के साथ उपयोग करना](/docs/tutorials/stateful-application/cassandra/)
+
+* [एक CP वितरित प्रणाली, ज़ूकीपर चलाना](/docs/tutorials/stateful-application/zookeeper/)
+
+## क्लस्टर
+
+* [AppArmor](/docs/tutorials/clusters/apparmor/)
+
+* [seccomp](/docs/tutorials/clusters/seccomp/)
+
+## सर्विस
+
+* [सोर्स आईपी का उपयोग करना](/docs/tutorials/services/source-ip/)
```

Comment:
> * [सोर्स आईपी का उपयोग करना](/docs/tutorials/services/source-ip/)

Would स्रोत be a better word to use than सोर्स?

Reply:
I think `Source IP` is a commonly used term using Hindi for Source may lead to confusions. isn't it?

Reply:
Makes sense, yes, when we approach it from that angle.